docs/source/courses/advanced/xarray.rst

```diff
@@ -3,9 +3,12 @@ Create ``xarray.DataArray`` from an IDS
 
 .. info::
 
-    In this lesson you will create a ``DataArray`` manually. In a future version of
-    IMAS-Python we plan to include functionality that will automatically do this for you.
-    That should further simplify working with data inside IDSs.
+    This lesson was written before :py:func:`imas.util.to_xarray` was
```

Comment:
Is there still value in keeping this lesson if it's completely superceded by the new capabilities introduced by this PR?

Reply:
Some of the examples can still be useful for people, so I would not remove it: there's some practical use cases for accessing IDS metadata, and some examples for working with IMAS data in Xarray Datasets.